mcmcfit.py

```diff
@@ -794,6 +816,7 @@ def ln_prob(parList,phi,y,e,width=None):
 
         # CV model
         ax1.plot(xf,yf)
+        ax1.set_title(output_plots[iecl])
```

Comment:
Don't add a title; it may look nice in your notebook, but you'd have to remove it to put this plot in a paper.